print-cc.sh

```diff
@@ -2,4 +2,7 @@ cc="`head -1 conf-cc`"
 systype="`cat systype`"
 
 cat warn-auto.sh
-echo exec "$cc" '-c ${1+"$@"}'
+echo exec "$cc" -D_GNU_SOURCE '-c ${1+"$@"}'
```

Comment:
This doesn't appear necessary on Linux (the test works with the typo fix), and obviously wasn't hurting things by not being there (since everybody has been using the manual definition in byte_rchr because of the typo in trymemchr.c). I'm also not sure this is the best place for it (though all the other places I can think of have issues too). Unless this is an actual problem for others, I think I'll skip this for now.